features/user_can_login.feature

```diff
@@ -0,0 +1,31 @@
+Feature: User can log in
+
+As a user
+In order to access my account
+I would like to be able log in
+
+Background:
+        Given I am on the login page
+        And the following user exist:
```

Comment:
Populating the DB is the first thing we want to do in a backgroud

Reply:
ok